lib/src/base-styles/_nx-card.scss

```diff
@@ -14,91 +14,62 @@
   border-radius: $nx-border-radius;
   box-sizing: border-box;
   margin-bottom: $nx-spacing-md;
```

Comment:
The ways the top and bottom margins of the cards and containers interact is probably not ideal.  Containers have a 16px top margin.  Cards have a 16px bottom margin.  These two margins do not collapse into each other for adjacent containers, resulting in a total space of 32px.  Then of course the space between rows of a single, multi-line container is only the 16px card bottom margin.  My recommendation: remove the bottom margin from the cards, add a `row-gap` to the container, and add a bottom margin to the container

Reply:
> The ways the top and bottom margins of the cards and containers interact is probably not ideal. Containers have a 16px top margin. Cards have a 16px bottom margin. These two margins do not collapse into each other for adjacent containers, resulting in a total space of 32px. Then of course the space between rows of a single, multi-line container is only the 16px card bottom margin. My recommendation: remove the bottom margin from the cards, add a `row-gap` to the container, and add a bottom margin to the container

Good suggestion: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/14e55445ecb4714a15553f7625a7549a8c9efe1a